build/common.go

```diff
@@ -215,6 +215,31 @@ func (Build) Debug() error {
 	cfg.EnableDebug = true
 	return buildBackend(cfg)
 }
+func (Build) DebugLinuxAMD64() error {
+	cfg := newBuildConfig("linux", "amd64")
+	cfg.EnableDebug = true
+	return buildBackend(cfg)
+}
+func (Build) DebugLinuxARM64() error {
+	cfg := newBuildConfig("linux", "arm64")
+	cfg.EnableDebug = true
+	return buildBackend(cfg)
+}
+func (Build) DebugDarwinAMD64() error {
+	cfg := newBuildConfig("darwin", "amd64")
+	cfg.EnableDebug = true
+	return buildBackend(cfg)
+}
+func (Build) DebugDarwinARM64() error {
+	cfg := newBuildConfig("darwin", "arm64")
+	cfg.EnableDebug = true
+	return buildBackend(cfg)
+}
+func (Build) DebugWindowsAMD64() error {
+	cfg := newBuildConfig("windows", "amd64")
+	cfg.EnableDebug = true
+	return buildBackend(cfg)
+}
```

Comment:
I think all of these are pretty self explanatory but can you add comments anyway? just to render those when listing targets:

```
$ mage -l    
Targets:
  build:backend                 build a production build for the current platform
  build:darwin                  builds the back-end plugin for OSX.
  build:darwinARM64             builds the back-end plugin for OSX on ARM (M1).
  build:debug                   builds the debug version for the current platform
  build:debugDarwinAMD64        
  build:debugDarwinARM64        
  build:debugLinuxAMD64         
  build:debugLinuxARM64         
  build:debugWindowsAMD64       
  build:generateManifestFile    generates a manifest file for plugin submissions
...
```

Reply:
will do!